.drone.yml

```diff
@@ -67,36 +67,23 @@ steps:
       event:
         - push
         - pull_request
-  - name: publish-alpha
+  - name: publish-docker
     image: plugins/docker
     settings:
-      dockerfile: docker/Dockerfile
       repo: publicawesome/stargaze
-      username:
-        from_secret: docker_username
-      password:
-        from_secret: docker_password
       tags:
         - 12.0.0-alpha.1
-    when:
-      event:
-        - push
-      branch:
-        - spoorrthi/remove-claim
-  - name: docker_release
-    image: plugins/docker
-    settings:
-      dockerfile: docker/Dockerfile
-      repo: publicawesome/stargaze
       username:
         from_secret: docker_username
       password:
         from_secret: docker_password
-      auto_tag: true
+      dockerfile: docker/Dockerfile
+      platforms: linux/arm64,linux/amd64
+      push: true
     when:
       event:
+        - push
         - tag
-
```

Comment:
we have different steps for final releases (tag) vs latest or alpha versions, let's keep both, if you want to trigger an image push you can temporarily set it to your branch and also perhaps we could ad `main`